### ISSUE TITLE
Schema prefix not fetched from query

### ISSUE THREAD

**Issue by odarriba** [mojotech/scrivener_ecto#93]
We were using Scrivener.Ecto version 2.2.0 until today, and passing schema prefix on the query was working like:

```elixir
query
|> Map.put(:prefix, tenant_name)
|> Repo.paginate(page: page_no)
```

But with the update to the latest version (2.6.0 as of today), the library has stopped respecting the prefix on the query, which is one of the methods used by Ecto to specify the DB schema of a query.

This is the expected behaviour if we look at #91 , but is this correct? Shouldn't it respect the prefix on the query if no one is passed on the `paginate` method options?

I can work in a patch for this, but I would like to be sure of the expected behaviour before.

**Comment by gkdp** [comment]
Please do. The new update broke my application. 😞

**Comment by drewolson** [comment]
Apologies, this should now be fixed in `2.7.0`.

**Comment by odarriba** [comment]
Thanks @drewolson! <3
